client/components/status/status.test.js

```diff
@@ -1366,14 +1366,9 @@ describe("<Status /> interactions", () => {
       disableLifecycleMethods: true,
     });
     await wrapper.instance().getUserRadiusSessions();
-    expect(prop.logout).toHaveBeenCalledWith(
-      {
-        HAS_DOCUMENT_COOKIE: true,
-        changeListeners: [],
-        cookies: {default_macaddr: "4e:ed:11:2b:17:ae"},
-      },
-      "default",
-    );
+    expect(prop.logout).toHaveBeenCalledWith(expect.any(Cookies), "default");
+    const cookiesArg = prop.logout.mock.calls[0][0];
+    expect(cookiesArg.cookies).toEqual({default_macaddr: "4e:ed:11:2b:17:ae"});
```

Comment:
The `Cookies` object returned by `react-cookie` now includes additional
internal properties and methods (i.e.., `defaultSetOptions`, `update`),
which caused strict `toHaveBeenCalledWith` checks to fail.

Refactored tests to assert `Cookies` instances using `expect.any(Cookies)`
and verified internal cookie values explicitly.